main.tf

```diff
@@ -1,16 +1,107 @@
+locals {
+  tags = merge(
+    var.tags,
+    {
+      terraform_module = basename(abspath(path.module))
+    }
+  )
+}
 
-module "tags" {
-  source  = "rhythmictech/tags/terraform"
-  version = "1.0.0"
+resource "aws_cloudwatch_log_group" "vpn" {
+  name_prefix       = "vpn-${var.name}"
+  retention_in_days = var.log_retention_days
```

Comment:
change to `cloudwatch_log_retention_days`?

Reply:
Yeah, sure